fetch/fetch.go

```diff
@@ -11,24 +11,110 @@ import (
 )
 
 var (
+	trueBool = true
+)
+
+type Fetcher struct {
 	client *ssm.SSM
 	paths  []string
 	tags   []string
+	debug  bool
+}
 
-	trueBool = true
-)
+// MustSetEnv retrieves SSM parameters for the given paths and tags and sets each
+// as env variables via os.Setenv.  This function will panic if any error occurs.
+// Set debug to true for debug logging to STDOUT (WARNING: any sensitive SSM param values will be logged).
+func MustSetEnv(paths, tags []string, debug bool) {
+	fetcher := NewFetcher(paths, tags, os.Getenv("SSM_REGION"), debug)
+	params, err := fetcher.FetchParams(paths, tags)
+	if err != nil {
+		panic(err)
+	}
 
-func init() {
```

Comment:
Switched to having explicit initialization via `NewFetcher()` as this double `init()` in main and fetch packages (both with global `client` variables) seemed confusing and potentially dangerous.